packages/drupal/custom/src/ContentHub.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\custom;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\graphql_directives\DirectiveArguments;
+
+/**
+ * Service to query content listings.
+ */
+class ContentHub {
+
+  /**
+   * The entity type manager, to query and load pages.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * ContentHub constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   The entity type manager, to query and load pages.
+   */
```

Comment:
Yet another suggestion. This whole doc comment is rather useless. Maybe let's get rid of it completely?

Reply:
PHPStan complained about them.

Reply:
Disable it 😛